python/paddle/v2/fluid/tests/test_sequence_slice_op.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+import numpy as np
+import sys
+from op_test import OpTest
+
+class TestSequenceSliceOp(OpTest):
+    def set_data(self):
+        self.init_test_case()
+        # only supprot one level LoD
+        x = np.random.random(self.x_dim).astype('float32')
+        lod = self.x_lod
+        offset = np.array(self.offset).flatten().astype("int64")
+        length = np.array(self.length).flatten().astype("int64")
+
+        self.inputs = {'X': (x, lod), 'Offset': offset, 'Length': length}
+        outs = [] #np.zeros((100, 3, 2)).astype('float32')
+        out_lod = [[0]]
+        out_lod_offset = 0
+        for i in range(len(offset)):
+            sub_x = x[lod[0][i] + offset[i]: lod[0]
+                      [i] + offset[i] + length[i], :]
+            out_lod_offset = out_lod_offset + len(sub_x)
+            outs.append(sub_x)
+            out_lod[0].append(out_lod_offset)
+        outs = np.concatenate(outs, axis=0)
```

Comment:
I think it is better that putting those code(line 19~25) into a function.

Reply:
Those codes are short and only use once.